app/db.py

```diff
@@ -10,7 +10,8 @@ def init_app(app):
 def get_conn_info():
     return {
         'user' : current_app.config["MYSQL_USER"],
-        'host' : current_app.config["MYSQL_HOST"],
+        # 'host' : current_app.config["MYSQL_HOST"],
+        'host' : "mysql",
```

Comment:
I think you can just edit the default `config/default.json` to set this up to work with this docker setup.

Reply:
I noticed this happens in a few other places, this also applies.